NuGetPackageVerifier.json

```diff
@@ -14,6 +14,7 @@
     "adx-nonshipping": {
         "rules": [],
         "packages": {
+            "Microsoft.AspNetCore.DiagnosticsViewPage.Sources": { },
```

Comment:
@Eilon `packages.csv`?  This was moved to Coherence repo (to make downloading the file easier).  Please update both Coherence-Signed and Coherence copies. We'll delete the one in Coherence-Signed soon.